docs/destinations/forwarding-events/google-tag-manager-server-side/amplitude-tag-for-gtm-ss/amplitude-tag-configuration/index.md

```diff
@@ -4,6 +4,12 @@ date: "2021-11-24"
 sidebar_position: 100
 ---
 
+:::tip Populating the Amplitude Session ID
+
+In order to populate the Session ID in Amplitude so that your events will be stitched into sessions correctly, your Snowplow events need to have the [`client_session` context entity](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracker-setup/initialization-options/index.md#session-context) attached. Then the Amplitude Tag will automatically populate the Amplitude Session ID based on the `firstEventTimestamp` of the session the event belongs to.
```

Comment:
I might be just me but it’s not obvious what the second sentence means. Could you explain a bit more why the Amplitude session is derived from a timestamp and not simply from `user_sessionid`?

Reply:
Great point @stanch ! The reason is not obvious at all, so added a sentence to explain why this is needed.

Reply:
Now I get it! Thanks :)